ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1235,28 +1235,47 @@ def test_sosreport_includes_healthcheck(self, create_logfile):
         )
         assert msg in cmd.stdout_text
 
+    def modify_perms_run_healthcheck(self, filename, modify_permissions):
+        """
+        Modify the ipa logfile permissions and run
+        healthcheck command to check the status.
+        """
+        modify_permissions(self.master, path=filename, mode="0644")
+        returncode, data = run_healthcheck(
+            self.master, "ipahealthcheck.ipa.files", failures_only=True
+        )
+        assert returncode == 1
+        assert len(data) == 1
+        assert data[0]["result"] == "WARNING"
+        assert data[0]["kw"]["path"] == filename
+        assert data[0]["kw"]["type"] == "mode"
+        assert data[0]["kw"]["expected"] == "0600"
```

Comment:
This can only test against files that are expected to be 0600. Perhaps pass in the expected permission as well?

Reply:
@menonsudhir do you agree with my suggestion?